src/sentry/static/sentry/app/components/assistant/guideDrawer.jsx

```diff
@@ -24,32 +28,67 @@ export default class GuideDrawer extends React.Component {
 
   render() {
     return (
-      <div>
-        <div className="assistant-drawer-title">
-          {this.props.guide.steps[this.props.step - 1].title}
-        </div>
-        <div className="assistant-drawer-message">
-          {this.props.guide.steps[this.props.step - 1].message}
-        </div>
-        <div>
-          {this.props.step < this.props.guide.steps.length ? (
-            <div>
-              <Button onClick={nextStep}>{t('Next')} &rarr;</Button>
-              <Button onClick={this.handleDismiss}>{t('Dismiss')}</Button>
-            </div>
-          ) : (
-            <div>
-              <p>{t('Did you find this guide useful?')}</p>
-              <Button onClick={() => this.handleUseful(true)}>
-                {t('Yes')} &nbsp; &#x2714;
+      <StyledAssistantContainer>
+        <StyledAssistantInputRow>
+          <CueIcon hasGuide={true} />
```

Comment:
We could also skip the icon in the expanded view (when the user is already a click or 2 deep and is aware of what's happening). No strong preference either way